recipes/fftw/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: fftw
+  version: 3.3.4
+
+source:
+  fn: fftw-3.3.4.tar.gz
+  url: http://www.fftw.org/fftw-3.3.4.tar.gz
+  md5: 2edab8c06b24feeb3b82bbb3ebf3e7b3
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+
+about:
+  home: http://www.fftw.org/
+  license: GPL
```

Comment:
The recipes need a `test` and `extra/maintainers` sections. Take a look at the example recipe in the repository.

Note that the maintainers must be the GitHub handle as that is used to create teams for the feedstock later on.